resources/views/community/components/forum/post-author-box.blade.php

```diff
@@ -1,5 +1,4 @@
 <?php
-use App\Site\Enums\Permissions;
```

Comment:
This is still used on line 28

Reply:
Definitely did not intend to do that.

It looks like php-cs-fixer autodeleted it, and [there's no way to stop it from doing that](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/3846#issuecomment-401324204) unless I use `Permissions` somehow in a `<?php ?>` block. The latest commit to the branch does that, but seems like a fairly annoying workaround.